docs/config/configuring-nodes-storage.rst

```diff
@@ -0,0 +1,240 @@
+.. _agent-yml:
+
+==========================================
+Configuring the Nodes and Storage Backends
+==========================================
+
+To start the agents on a node, a configuration file must exist on the node at ``/etc/flocker/agent.yml``.
+The file must always include ``version`` and ``control-service`` items similar to these:
+
+.. code-block:: yaml
+
+   "version": 1
+   "control-service":
+      "hostname": "${CONTROL_NODE}"
+      "port": 4524
+
+The value of the hostname field should be a hostname or IP that is routable from all your node agents.
+
+When configuring node agents, consider whether the control service location you choose will have multiple possible addresses, and ensure the hostname you provide is the correct one.
+You should never choose ``127.0.0.1`` or ``localhost`` as the hostname, even if the control service is on same machine as the node agent.
```

Comment:
Oh, I see this is just a move. Well, please add a reason if you know of one. Otherwise, comment here saying that you're not going to.